src/modules/mod.rs

```diff
@@ -273,6 +275,7 @@ pub fn description(module: &str) -> &'static str {
         "package" => "The package version of the current directory's project",
         "perl" => "The currently installed version of Perl",
         "php" => "The currently installed version of PHP",
+        "pijul" => "The current channel of the repo in the current directory",
```

Comment:
Maybe it should be called `pijul_branch` in case someone wants to do a `status` module in the future.